checks/__init__.py

```diff
@@ -286,6 +300,9 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         self.name = name
         self.init_config = init_config or {}
         self.agentConfig = agentConfig
+        self.in_developer_mode = agentConfig.get('developer_mode') and psutil is not None
+        self.stats = None
```

Comment:
Same here, you should use a name that have less chances to conflict with existing checks